mkdocs_awesome_pages_plugin/navigation.py

```diff
@@ -37,6 +37,10 @@ def __init__(self, filename: str):
         )
 
 
+class VSection(Section):
```

Comment:
Could you spell this out as `VirtualSection`? Please also do so for any other instances of the term.